### ISSUE TITLE
password field not rendering

### ISSUE THREAD

**Issue by ilyes-kechidi** [netojose/laravel-bootstrap-4-forms#18]
inputs fields of type password are not rendering

**Comment by ilyes-kechidi** [comment]
i managed to fix this issue just by adding
```
    /**
     * Return a password input tag
     *
     * @return string
     */
    public function password(): string
    {
        return $this->_renderInput('password');
    }
```

to the **FormBuilder** class and then adding `'password'` to `$_allowedRenders` array in `FormService` 

i can send you a pull request

**Comment by netojose** [comment]
Yes @ilyes-kechidi , this is the correct fix. Please, send a pull request, I will merge and create a new version `2.0.1` with your fix.

Thankyou!

**Comment by netojose** [comment]
Fixed on 2.0.1 version. Thanks to @ilyes-kechidi
